tests/utils/test_qwen_tts_config.py

```
@@ -0,0 +1,213 @@
+        project_root = os.path.dirname(
+            os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+        )
+        script_path = os.path.join(project_root, "scripts", "qwen_tts_api.py")
+        print(f"Project root: {project_root}")
+        print(f"Script path: {script_path}")
+        print(f"Script exists: {os.path.exists(script_path)}")
+
+        if not os.path.exists(script_path):
+            pytest.skip("qwen_tts_api.py script not found")
+
+        proc = subprocess.Popen(
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)